domains/morsteo.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Link to my portfolio site.",
+  "repo": "https://github.com/abh80/abh80.github.io",
+  "owner": {
+    "username": "abh80",
+    "email": "boatgithub27@gmail.com",
+    "twitter": "trackerstars"
+  },
+  "record": {
+    "CNAME": "abh80.github.io"
```

Comment:
pretty sure this cname have been poined to another website already, so this cname cant be reused since github only allows 1 custom domain per page.